src/petter_hetland_ex/ex04/myrand.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Petter Hetland'
+__email__ = 'pehe@nmbu.no'
+
+
+class LCGRand:
+    """
+    This class is a linear congruential generator (LCG),
+    which generates numbers according to the following equation
+
+
+    r[n+1] = a * r[n] mod m
+
+    where ``a = 7**5 = 16807`` and ``m = 2**31-1``.
+    """
+    def __init__(self, seed):
+        self.init_number = seed
+
+    def rand(self):
+        self.next_number = (16807 * self.init_number) % (2**31-1)
```

Comment:
we use self.variable for variable defined in _init_ method, here you can use next_number as a name